src/libs/Renderer.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable import/no-cycle */
+import Behaviors from './Behaviors.js';
+import Storage from './Storage.js';
+
+export default class Renderer {
+  constructor(selector = '#test') {
+    this.selector = selector;
+    this.element = document.querySelector(this.selector);
+    this.syncUpdates();
+    this.render();
+  }
+
+  syncUpdates(payload = []) {
+    this.list = new Storage('todo').get() || payload;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  serialUpdates() {
+    const updated = [];
+    const currentList = document.querySelectorAll('#list li');
+    // eslint-disable-next-line no-restricted-syntax, guard-for-in
+    for (const it in currentList) {
+      updated.push(this.list[Number(currentList[it].dataset?.index)]);
+    }
+    // updated = updated.slice(0, currentList.length);
```

Comment:
leaving old codes as inline comments are not advisable so as not to make your code look messy.